CMakeLists.txt

```diff
@@ -698,20 +704,10 @@ if ( enable-threads )
 endif ( enable-threads )
 
 unset ( HAVE_OPENMP CACHE )
-find_package ( OpenMP QUIET )
+find_package ( OpenMP 4.0 QUIET )
```

Comment:
Carefully with this one. I remember that often it doesn't specify `OpenMP_C_VERSION`. Instead it uses `OpenMP_C_SPEC_DATE`. We will have to review all the CI builds and see whether openMP is still found properly.

Reply:
FindOpenMP.cmake has been always distributed with the CMake program, and was seriously buggy sometimes.

https://cliutils.gitlab.io/modern-cmake/chapters/packages/OpenMP.html

Shouldn't be a problem in cmake 3.13 anymore.